OPENSTACK-2769: change capacity config for capacity calculation

zhang-shengping opened the pull request.

zhaoqin-github reviewed (changes requested):
This change is much more complex than I predict. I require very long time to review.

zhaoqin-github commented on f5lbaasdriver/v2/bigip/device_scheduler.py:
```diff
@@ -287,23 +292,34 @@ class CapacityFilter(DeviceFilter):
             for lic in capacity_const["license"]:
                 lic_const = capacity_const["license"][lic]
                 # Negative value means unlimited
-                for key in ["bandwidth", "lod", "rod", "cod"]:
```
We cannot remove "bandwidth", because the inherited BandwidthCapacityFilter requires it.

zhaoqin-github commented on f5lbaasdriver/v2/bigip/constants_v2.py:
Need to keep osr and bandwidth value, because BandwidthCapacityFilter need them when handling unknown bandwidth devices

Commit:
OPENSTACK-2769: move cfg OPTs to config.py

Commit:
OPENSTACK-2769: change capacity config for capacity calculation

The capacity default value in scheduler.json is for all unknown
device.

License of hardware is variety, but they are not start with 'VE'.

We only have some constant value for VE and unknown device.

hardware should be configure by user in scheduler.json.

zhaoqin-github commented on f5lbaasdriver/v2/bigip/config.py:
```diff
@@ -0,0 +1,94 @@
+            default=[
+                        'AvailabilityZoneFilter',
+                        'FlavorFilter',
+                        'BandwidthCapacityFilter',
```
Need to adjust filter list
